homes/development/default.nix

```diff
@@ -5,6 +5,7 @@
 {
   imports = [
     ./direnv.nix
+    ./editor.nix
```

Comment:
is editor.nix really the best name for this? how about `zed.nix` (to match helix, and provide a separate space for global declarative config) or `collaboration.nix` (for our purpose)